### DISCUSSION TITLE
Please add a hotkey for toggle sidebar focus in Nemo

### DISCUSSION THREAD

**Discussion by Dan-cer** [linuxmint/discussions#421] [Pull requests]
All people that want to use keyboard only instead of mouse often need to get to Nemo sidebar by using a hotkey.
At the moment it isn't more than a workaround, using 
1. F3 and
2. F6.
I know that it is possible to customize Nemo hotkeys in ~/.gnome2/accel/nemo (self-created). However, for access sidebar and back there is no option, right?
So please, implement it for Nemo in next future!
Thank you very much!

**Comment by jeffreyesun** [comment]
This would be very useful to me also. I will add that with nemo-terminal, F6 focuses the terminal instead of the sidebar, then hitting F6 again writes a tilde to the terminal instead of changing focus. So with nemo-terminal, there is apparently no way to focus the sidebar using the keyboard.
